### PR TITLE
Update to v1.5.0 and add ppc64le

### PR DESCRIPTION
**Pull request by matthiasdiener**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Closes #13 #14 #15

### CODE REVIEW

**Comment by matthiasdiener** [comment]
Ready for review @conda-forge/libunwind

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,11 +26,11 @@ requirements:
 
 test:
   commands:
-    - test -f "${PREFIX}/lib/libunwind.so"
-    - test ! -f "${PREFIX}/lib/libunwind.a"
+    - test -f "${PREFIX}/lib/libunwind.so"  # [x86_64] 
+    - test -f "${PREFIX}/lib64/libunwind.so"  # [ppc64le]
```
Comment (xhochy) [inline recipe/meta.yaml]: This is wrong, shared libraries should always go into `lib` on conda-forge.

File: recipe/build.sh [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+
+
 # First call only the install target as compilation for tests only works with an installed libunwind
 make install -j${CPU_COUNT}
 make -j${CPU_COUNT}
-make check -j${CPU_COUNT}
+
+
+# The tests are known to be flakey so allow them to fail
+make check || true
```
Comment (xhochy) [inline recipe/build.sh]: Then what is the sense of running them at all?

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipe/build.sh (hunk 2 above)
Comment (matthiasdiener) [reply recipe/build.sh]: Not sure, I took this from a previous PR (#15). I can disable the tests if you prefer.

Edit:
Just to clarify, the tests do indeed fail, even on x86_64.

Comment (xhochy) [reply recipe/build.sh]: If this is really due to flakiness, let's get rid of them completely. If we don't check them, then we don't need to waste the CPU time for them.

File: recipe/meta.yaml (hunk 1 above)
Comment (matthiasdiener) [reply recipe/meta.yaml]: Should be fixed now.

**Review by xhochy — APPROVED** [review]

**Pull request merged**